app/src/main/java/org/p2p/wallet/auth/model/Username.kt

```diff
@@ -3,6 +3,5 @@ package org.p2p.wallet.auth.model
 data class Username(
     val value: String,
     val domainPrefix: String,
-    // FIXME: New version of name service returns domain as well. We should remove domainPrefix asap
     val fullUsername: String = value + domainPrefix
```

Comment:
we better remove this and `domainPrefix` fields. 
All usernames will come from backend. 
And looks like we need to remove feature toggle for `username_value`
Please workaround

Reply:
done!
checked on main screen, search fragment

Reply:
I can't see, if it's done. 
You are still concatenating here: `value + domainPrefix`